### PR TITLE
Outfits view

### PR DESCRIPTION
**Pull request by Cristina-Figueroa**
1) Fixed the render issue, ya se puede ver en Android.
2) Overall styling of Outfits Main Page, Edit page and Make Your Outfit page, falta la logica

### CODE REVIEW

**Review by Adhel-Esteban1 — CHANGES_REQUESTED** [review]
Hi, Cristina. I reviewed the code, and tested it on android it works fine. On iOS there's a bit of trouble with the styles though. But I suggest a big change, even bigger than restyle for iOS. Reading and implementing code for over 500 lines and filtering what is necessary and unnecessary is a complicated task. I suggest a refactor of the 3 changed files: outfits.tsx, addOutift.tsx, and edit.tsx. 

There are unused imports, unused functions, constants, commented code, and more. This makes the code very difficult to review, and understand what's going on.

Here's a plan idea. I managed to shorten the 500+ line code of outfits.tsx file into a simple and easy-to-read 45-line code on the branch 120-seamless-design. Right now there's a PR for that branch under review, and we can merge that one first. Then, I can help out in the refactoring of the 3 files I mentioned.

**Comment by Cristina-Figueroa** [comment]
Hi Adhel! I've done the refactoring you've requested for the three outfit files, outfits.tsx, addOutfit.tsx, and edit.tsx. Right now addOutfit.tsx and edit.tsx average on 230 lines of code, while outfits.tsx has about 180 lines of code. I've created about 7 files of components used by all three files. They are located under the components folder of the project. 

Right now, there's only styling for android version of the app, which means I'm still gonna need help styling the iOS part for the outfits screens/pages. Other than that, we can start working on the implementation of features as well as add your part of the MainOutfit page.

Let me know if you need any help understanding the code!

File: components/EditOutfitButton.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    return(
+      <View style={{paddingBottom:5}}>
+
+      {/* Separator */}
+        <View style={styles.separator}/>
+  
+      {/* Button */}
+        <View style={styles.OutfitButton}> 
+          <Pressable  
+            // onPress={() => setModalVisible(true)}
```
Comment (Adhel-Esteban1) [inline components/EditOutfitButton.tsx:20]: Commented code. But I'm unsure if it's necessary.

File: components/OutfitsFilterBoxes.tsx [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+      },
+  }),
+  },
+
+  fixedContainer: {
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    // backgroundColor: 'lightblue',
```
Comment (Adhel-Esteban1) [inline components/OutfitsFilterBoxes.tsx:123]: Commented code can be removed.

File: components/OutfitsFilterBoxes.tsx [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    ...Platform.select({
+      ios: {
+          height: "20%",
+          width: "95%",
+      },
+      android: {
+          height: "25%",
+          width: "100%",
+          // top:5
```
Comment (Adhel-Esteban1) [inline components/OutfitsFilterBoxes.tsx:132]: Commented code can be removed.

File: components/OutfitsFilterBoxes.tsx [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+    flexDirection: 'column',
+    backgroundColor: "#F0F0F0", 
+    margin: 10, 
+    marginBottom:0
+    // backgroundColor: 'black', //ffff
+  },
+    
+  smallerContainer: { //all buttons are inside this smaller container
+    flex: 1,
+    // backgroundColor: 'pink',
```
Comment (Adhel-Esteban1) [inline components/OutfitsFilterBoxes.tsx:99]: Commented code can be removed.

File: components/OutfitsFilterBoxes.tsx [hunk 5]
```diff
@@ -0,0 +1,236 @@
...
+// Styles
+const styles = StyleSheet.create({
+  buttonContainer: { //all components are in this parent container
+    flex: 1,
+    justifyContent: 'space-around',
+    flexDirection: 'column',
+    backgroundColor: "#F0F0F0", 
+    margin: 10, 
+    marginBottom:0
+    // backgroundColor: 'black', //ffff
```
Comment (Adhel-Esteban1) [inline components/OutfitsFilterBoxes.tsx:94]: Commented code can be removed.

File: components/OutfitButton.tsx [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    );
+  }
+
+
+// Styles
+const styles = StyleSheet.create({
+    
+    outfitsButtonContainer:{ // all outfits buttons are inside this container
+        flex: 1,
+        // backgroundColor: 'lightblue',
```
Comment (Adhel-Esteban1) [inline components/OutfitButton.tsx:35]: Commented code can be removed.

File: components/OutfitButton.tsx [hunk 7]
```diff
@@ -0,0 +1,79 @@
+import React, {useState} from 'react';
+import { Pressable, StyleSheet, Platform,} from 'react-native';
```
Comment (Adhel-Esteban1) [inline components/OutfitButton.tsx:2]: Unused import can be removed.

File: components/NameOfOutfitPlaceholder.tsx
Comment (Adhel-Esteban1) [inline components/NameOfOutfitPlaceholder.tsx:1]: Perhaps the entire lower part that houses the buttons and the rectangle can be a single component.

File: components/EditOutfitButton.tsx [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+            >
+        <View style={{flex: 1,
+            flexDirection: 'row', 
+            alignItems:'center', 
+            justifyContent:'space-between',
+            backgroundColor: '#F0F0F0' //'#F194FF'
```
Comment (Adhel-Esteban1) [inline components/EditOutfitButton.tsx:26]: Commented code can be removed.

File: components/EditOutfitButton.tsx [hunk 9]
```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Pressable, StyleSheet, Platform} from 'react-native';
+
```
Comment (Adhel-Esteban1) [inline components/EditOutfitButton.tsx:3]: Unused import can be removed.

File: app/addOutfit.tsx [hunk 10]
```diff
@@ -1,161 +1,227 @@
...
+        backgroundColor: '#F0F0F0' ,//'#D9D9D9'
+    },
+    filterTitleText:{
+        fontWeight:'bold'
+    },
+
+    // Name of Outfit and Save&CancelButton Container
+    nameOfOutfitContainer:{
+        position: 'absolute',
+        // top: 0,
```
Comment (Adhel-Esteban1) [inline app/addOutfit.tsx:174]: Commented code can be removed.

File: app/addOutfit.tsx [hunk 11]
```diff
@@ -1,161 +1,227 @@
...
 
-    separator: { //added this to separate each button a smidge, is added just before a new button
-        marginVertical: 20,
-        height: 10,
-        width: '3%',
+
+    // Filter Boxes
+    filterTitle:{
+        alignItems: 'center',
+        backgroundColor: '#F0F0F0' ,//'#D9D9D9'
```
Comment (Adhel-Esteban1) [inline app/addOutfit.tsx:165]: Commented code can be removed.

File: app/addOutfit.tsx [hunk 12]
```diff
@@ -1,161 +1,227 @@
...
+        
+        maxHeight: 350,
+        marginTop: 75,
+        marginBottom: 120,
+
+        ...Platform.select({
+            // ios: {
+            //     height: "20%",
+            //     width: "95%",
+            // },
```
Comment (Adhel-Esteban1) [inline app/addOutfit.tsx:151]: Commented code can be removed.

File: app/addOutfit.tsx [hunk 13]
```diff
@@ -1,161 +1,227 @@
...
+            android: {
+                // height: "120%",
```
Comment (Adhel-Esteban1) [inline app/addOutfit.tsx:153]: Commented code can be removed.

File: app/addOutfit.tsx [hunk 14]
```diff
@@ -1,161 +1,227 @@
...
+        elevation:2, 
+    },
+    
+    outfitsContainer:{ //container that previews the outfits
         flex: 1,
-        backgroundColor: 'white',
-        padding: 25, 
-        margin: 10, //margin of the square - how big is it
-        borderRadius: 20, //rounds edges
+        // backgroundColor: 'pink',
```
Comment (Adhel-Esteban1) [inline app/addOutfit.tsx:138]: Commented code can be removed.

File: app/addOutfit.tsx [hunk 15]
```diff
@@ -1,161 +1,227 @@
...
-        margin: 10, //margin of the square - how big is it
-        borderRadius: 20, //rounds edges
-        paddingHorizontal:10,
-        paddingVertical: 10,
-        justifyContent:'flex-end',
-        // alignContent:'flex-end',
-        flexDirection: 'column',
-        // minHeight: '10%', 
-        // maxWidth: '80%'
+        // backgroundColor: 'pink',
```
Comment (Adhel-Esteban1) [inline app/addOutfit.tsx:105]: Commented code can be removed.

File: app/addOutfit.tsx [hunk 16]
```diff
@@ -1,161 +1,227 @@
...
 
 const styles = StyleSheet.create({
-    container: { //all smaller containers are inside this big container
+    bigContainer: { //all smaller containers are inside this big container
         flex: 1,
-        // alignItems: 'center',
-        // justifyContent: 'center',
-        // backgroundColor: 'pink',
-
+        // backgroundColor: 'black',
```
Comment (Adhel-Esteban1) [inline app/addOutfit.tsx:101]: Commented code can be removed.

File: app/(tabs)/outfits.tsx [hunk 17]
```diff
@@ -1,192 +1,180 @@
...
+    borderRadius: 10, 
+    marginVertical: 1,
   },
 
-  
-});
+  // Outfit Buttons container
+  outfitsButtonContainer:{ // all outfits buttons are inside this container
+    flex: 1,
+    // backgroundColor: 'lightblue',
```
Comment (Adhel-Esteban1) [inline app/(tabs)/outfits.tsx:162]: Commented code can be removed.

File: app/(tabs)/outfits.tsx [hunk 18]
```diff
@@ -1,192 +1,180 @@
...
-    marginVertical: 10,
-    height: .5,
-    width: '80%',
-    backgroundColor: 'white'
+  fixedContainer: {
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    right: 0,
+    // backgroundColor: 'lightblue',
```
Comment (Adhel-Esteban1) [inline app/(tabs)/outfits.tsx:107]: Commented code can be removed.

**Review by Adhel-Esteban1 — CHANGES_REQUESTED** [review]
Most of the changes I suggest is removing commented code, and some unused imports. I learned this recently and it really helps for other developers. 

I think using the components for buttons like cancel and more is a very good idea. However, I think it may have been a little overdone. I think perhaps the entire lower square could've been a component, and likewise, restyle the existing filter component and use that component on the top filters. Seamless design can fix this later though. 

Other than that, the code is much more readable. I think the many components was overdone as it's not looking good on iOS, but that can be fixed later. The foundation is set for future development. 

Just check out the comments on unused imports and commented code, and that should be good to go.

**Review by Adhel-Esteban1 — APPROVED** [review]
Very well done! Much better now! And much easier to read as well!

**Pull request merged**